assignments/objects.js

```diff
@@ -11,54 +11,109 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const Mitzi = Object.create(example);
+Mitzi.id = 1;
+Mitzi.name = "Mitzi";
+Mitzi.email = "mmelloy0@psu.edu";
+Mitzi.gender = "F";
+
+const Kennan = Object.create(example);
+Kennan.id = 2;
+Kennan.name = "Kennan";
+Kennan.email = "kdiben1@tinypic.com";
+Kennan.gender = "M";
+
+const Keven = Object.create(example);
+Keven.id = 2;
+Keven.name = "Keven";
+Keven.email = "kmummery2@wikimedia.org";
+Keven.gender = "M";
+
+const Gannie = Object.create(example);
+Gannie.id = 3;
+Gannie.name = "Gannie";
+Gannie.email = "gmartinson3@illinois.edu";
+Gannie.gender = "M";
+
+const Antonietta = Object.create(example);
+Antonietta.id = 4;
+Antonietta.name = "Antonietta";
+Antonietta.email = "adaine5@samsung.com";
+Antonietta.gender = "F";
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(Mitzi.name);
 
 // Kennan's ID
-
+console.log(Kennan.id);
 // Keven's email
+console.log(Keven.email);
 
 // Gannie's name
-
+console.log(Gannie.name);
 // Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
+console.log(Antonietta.gender);
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+Kennan.speak = function() {
+  return `Hello, my name is ${this.name}!`;
+};
+console.log(Kennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+Antonietta.multiplyNums = (num1, num2) => num1 * num2;
+console.log(Antonietta.multiplyNums(3, 4));
 
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
-// ==== Stretch Challenge: Nested Objects and the this keyword ==== 
+// ==== Stretch Challenge: Nested Objects and the this keyword ====
```

Comment:
great job on the stretch. I like seeing it when people push themselves. Keep this up